plugins/file/file_output.go

```diff
@@ -257,11 +296,15 @@ func (o *FileOutput) committer(or OutputRunner, wg *sync.WaitGroup) {
 				// Channel is closed => we're shutting down, exit cleanly.
 				break
 			}
+			if err = o.tryRotateFile(); err != nil {
+				panic(fmt.Sprintf("FileOutput unable to rotate file '%s': %s",
```

Comment:
Actually, neither panic nor LogError is ideal here, what *should* happen is that the plugin should exit, returning an error from the Run method. Same is true for the panic that's already in there. Doing this will require restructuring the plugin a bit.